server/portal/apps/auth/api/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import re_path
+from portal.apps.auth.api.views import TapisToken
+
+
+app_name = 'auth_api'
+urlpatterns = [
+    re_path('tapis', TapisToken.as_view(), name='tapis_token'),
```

Comment:
Can use `path` here instead of `re_path`

Reply:
yes, fixed 👍